docs/xxcontinuationcache.md

```diff
@@ -29,10 +29,10 @@
 
         -XX:ContinuationCache:t1=<size>,t2=<size>
 
-| Setting                 | Value      | Default                     |
-|-------------------------|------------|:---------------------------:|
-| `t1=<size>`    |  0-2^32 (4,294,967,295) |  1                       |
-| `t2=<size>`    |  0-2^32 (4,294,967,295) |  Number of processors x 2                       |
+| Setting | Value | Default |
+|---------|-------|:-------:|
+| `t1=<size>` | 0 to (2<sup>32</sup>-1) (4,294,967,295) | 1 |
```

Comment:
There needs to be a space at least before the `-`, it doesn't look good.
https://eclipse-openj9.github.io/openj9-docs-staging/1172/xxcontinuationcache/

Reply:
Added space before and after `-`.

Reply:
The spaces were removed during generation: https://eclipse-openj9.github.io/openj9-docs-staging/1172/xxcontinuationcache. Is there a way to prevent their removal?

Reply:
Please wait until the staging job is complete so I can see what it looks like.
If it's anything like the formatting here, I will be happy with either 2<sup>32</sup>-1 (no spaces) or 2<sup>32</sup> - 1 (with spaces), with a slight preference for the former.